mmv1/products/appengine/StandardAppVersion.yaml

```diff
@@ -73,6 +73,19 @@ examples:
       org_id: 'ORG_ID'
     ignore_read_extra:
       - 'delete_service_on_destroy'
+    app_engine_bundled_services:
+      - 'app_identity_service'
+      - 'blobstore'
+      - 'capability_service'
+      - 'datastore_v3'
+      - 'images'
+      - 'mail'
+      - 'memcache'
+      - 'modules'
+      - 'search'
+      - 'taskqueue'
+      - 'urlfetch'
+      - 'user'
```

Comment:
```suggestion
```

For details on how to add a test. please see https://googlecloudplatform.github.io/magic-modules/test/test/#add-a-create-test